site/app/controllers/MiscController.php

```diff
@@ -427,17 +416,7 @@ public function downloadSubmissionZip($gradeable_id, $user_id, $version, $is_ano
                     $file_path = $file->getRealPath();
                     $relative_path = substr($file_path, strlen($path) + 1);
                     if ($this->core->getAccess()->canI("path.read", ["dir" => $folder_name, "path" => $file_path, "gradeable" => $gradeable, "graded_gradeable" => $graded_gradeable, "gradeable_version" => $gradeable_version->getVersion()])) {
-                        // For scanned exams, the directories get polluted with the images of the split apart
-                        // pages, so we selectively only grab the PDFs there. For all other types,
-                        // we can grab all files regardless of type.
-                        if ($gradeable->isScannedExam()) {
-                            if (mime_content_type($file_path) === 'application/pdf') {
-                                $zip_stream->addFileFromPath($folder_name . "/" . $relative_path, $file_path);
-                            }
-                        }
-                        else {
-                            $zip_stream->addFileFromPath($folder_name . "/" . $relative_path, $file_path);
-                        }
+                        $zip_stream->addFileFromPath(FileUtils::joinPaths($folder_name, $relative_path), $file_path);
```

Comment:
Why now download all the images? Can't we still check if it's a bulk gradeable?

Reply:
Back when this PR was initially made I think this was discussed and we decided to remove this to remove potential confusion with files being silently removed in the backend but given the size of images in the zip it might be good to bring this up to see if we want to change this behavior.